### ISSUE TITLE
Multiselect fields are useless if you have too much records/options

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#2987]
Hi, i start by example, where i show you my "Resource calendar" of Projects

![crm_projekt_resource](https://cloud.githubusercontent.com/assets/5773777/22330049/85d1abe4-e3c4-11e6-9dc4-dc743c82c276.png)

It's very hard to choose one of project or users if i have more as 1300 projects yet, or more as 10 employees (or part time workers).

Same problem is in other multiselect fields in filters or in modules. If you have more as 5 options you are starting lost yourself by searching in currently multiselect fields 

I found something like this and we used it on other project (not into suitecrm)

http://davidstutz.github.io/bootstrap-multiselect/#post
![example](https://cloud.githubusercontent.com/assets/5773777/22330168/25c2881c-e3c5-11e6-88f3-7903cd955239.png)

We love this, because you can search and choosing your options in multiselect field in one enviroment.

Please feedback my suggestion / enhancement because the SuiteCRM is the best free CRM system, but still are some things which need to be done for best UX experience :)

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Multiselect fields are useless if you have too much records/options](https://trello.com/c/qqr3fYH6/38-multiselect-fields-are-useless-if-you-have-too-much-records-options)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
